hypervisor/arch/x86/trusty.c

```diff
@@ -7,7 +7,8 @@
 #include <hypervisor.h>
 #include <hkdf.h>
 
-#define TRUSTY_VERSION 1
+#define TRUSTY_VERSION   1
+#define TRUSTY_VERSION_2 2
```

Comment:
version is uint32_t, right? use 1U if it is.

Reply:
Fixed in latest version.